app/views/smart_answers/landing.html.erb

```diff
@@ -38,4 +38,10 @@
       <% end %>
     </article>
   </div>
+
+  <div class="govuk-grid-column-one-third govuk-!-margin-top-9">
+    <% if @presenter.show_escape_link? %>
+      <%= render "smart_answers/escape-link" %>
+    <% end %>
+  </div>
```

Comment:
I don't think we need this - it won't be rendered on the start page. Also in production the start page is rendered by [frontend](https://github.com/alphagov/frontend), so this won't actually change how the start page looks live.

Reply:
Hi @theseanything - resolved - thanks 👍